src/main/java/core/basesyntax/service/ParseData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ParseData<T> {
```

Comment:
Why do you need a generic here?
```suggestion
public interface ParserData {
```